src/Symfony/Component/Notifier/Bridge/Mobyt/MobytTransport.php

```diff
@@ -31,21 +31,25 @@ final class MobytTransport extends AbstractTransport
     private $accountSid;
     private $authToken;
     private $from;
-    private $typeQuality;
+    private $messageType;
```

Comment:
I think the new name is actually worse than the old one considering that all values are about quality (at least, that's what is implemented in the validation of the values). If we really want to rename it, why not just `quality` or `quality_level`?

Reply:
I thought the same, but want to stick to their documentation where they name it like this. For me it is less confusing because we neither need to map the parameter nor explain it somewhere

Reply:
Then I think I prefer to not change the current name. If we rename, we want something more meaningful, not less meaningful. In any case, I don't think spending so much time on this is needed. So, I would vote to keep it as is and close.

Reply:
Yes, but right now we have `type_quality` in the DSN and `message_type` in the options and the constants, so this at least uses the same wording across all code parts

Reply:
@fabpot I removed the renaming BC break